pages.es/common/git-commit.md

```diff
@@ -5,15 +5,19 @@
 
 - Realiza una confirmación de los archivos marcados al repositorio con un mensaje:
 
-`git commit -m "{{mensaje}}"`
+`git commit --message "{{mensaje}}"`
 
 - Realiza una confirmación de los archivos marcados con un mensaje leído desde un archivo:
 
 `git commit --file {{ruta/al/archivo_con_mensaje_de_la_confirmación}}`
 
 - Marca automáticamente todos los archivos modificados y realiza una confirmación con un mensaje:
 
-`git commit -a -m "{{mensaje}}"`
+`git commit --all --message "{{mensaje}}"`
+
+- Confirma todos los archivos preparados y firmalos con una llave de GPG (o la que este definida en el archivo de configuración si no se especifica un argumento):
```

Comment:
Why `los firma` rather than `firmalos`?

Reply:
`Firmalos` it is not the correct way to express what is described in Spanish. `los firma` is in the right way in terms of syntax & expression

Reply:
`Firmalos` is indeed incorrect, it is missing an accent (`Fírmalos`). However, [it seems possible to write `Fírmalos`](https://www.rae.es/dpd/pronombres%20personales%20%C3%A1tonos) (see 3c).

Reply:
I am also not sure about writing `este` with an accent, I will rewrite the phrase to indicate `llave` is **in** the configuration rather than [**near**](https://www.diccionariodedudas.com/este-este-o-este)

Reply:
perfect!

Reply:
I did not mean to cause controversy @kant, I retract my argument and will use `los firma`

Reply:
Perfect, Sir!

Reply:
> Perfect, Sir!

No, thank your for patience.

Reply:
A pleasure, Sir. Keep working hard for the benefit of humankind!!